routes/stats.js

```diff
@@ -47,4 +47,62 @@ statsRouter.get('/volunteer/:volunteerId', async (req, res) => {
   }
 });
 
+statsRouter.get('/week', async (req, res) => {
+  try {
+    const response = await pool.query(
+      `SELECT SUM(pounds) AS pound_sum, SUM(ounces) AS ounces_sum FROM event_data INNER JOIN events on event_data.event_id=events.id WHERE events.date BETWEEN CURRENT_DATE - INTERVAL '7 days' AND CURRENT_DATE`,
+    );
+    const pounds = parseFloat(response.rows[0].pound_sum);
+    const ounces = parseFloat(response.rows[0].ounces_sum);
+    const total = (pounds + ounces / 16).toFixed(2);
+    res.json(total);
+  } catch (err) {
+    res.status(400).json(err);
+  }
+});
+
+statsRouter.get('/year', async (req, res) => {
+  try {
+    const response = await pool.query(
+      `SELECT SUM(pounds) AS pound_sum, SUM(ounces) AS ounces_sum FROM event_data INNER JOIN events on event_data.event_id=events.id WHERE events.date BETWEEN CURRENT_DATE - INTERVAL '1 year' AND CURRENT_DATE`,
+    );
+    const pounds = parseFloat(response.rows[0].pound_sum);
+    const ounces = parseFloat(response.rows[0].ounces_sum);
+    const total = pounds + ounces / 16;
+
+    const responseTwo = await pool.query(
+      `SELECT SUM(pounds) AS pound_sum, SUM(ounces) AS ounces_sum FROM event_data INNER JOIN events on event_data.event_id=events.id WHERE events.date BETWEEN CURRENT_DATE - INTERVAL '2 years' AND CURRENT_DATE - INTERVAL '1 year'`,
+    );
+    const poundsTwo = parseFloat(responseTwo.rows[0].pound_sum);
+    const ouncesTwo = parseFloat(responseTwo.rows[0].ounces_sum);
+    const totalTwo = poundsTwo + ouncesTwo / 16;
+    const rate = (((totalTwo - total) / total) * 100).toFixed(2);
+    res.json(rate);
+  } catch (err) {
+    res.status(400).json(err);
+  }
+});
+
+statsRouter.get('/month', async (req, res) => {
+  try {
+    const response = await pool.query(
+      `SELECT SUM(pounds) AS pound_sum, SUM(ounces) AS ounces_sum FROM event_data INNER JOIN events on event_data.event_id=events.id WHERE events.date BETWEEN CURRENT_DATE - INTERVAL '1 month' AND CURRENT_DATE`,
+    );
+    const pounds = parseFloat(response.rows[0].pound_sum);
+    const ounces = parseFloat(response.rows[0].ounces_sum);
+    const total = pounds + ounces / 16;
+
+    const responseTwo = await pool.query(
+      `SELECT SUM(pounds) AS pound_sum, SUM(ounces) AS ounces_sum FROM event_data INNER JOIN events on event_data.event_id=events.id WHERE events.date BETWEEN CURRENT_DATE - INTERVAL '3 months' AND CURRENT_DATE - INTERVAL '1 month' `,
+    );
+    const poundsTwo = parseFloat(responseTwo.rows[0].pound_sum);
+    const ouncesTwo = parseFloat(responseTwo.rows[0].ounces_sum);
+    const totalTwo = poundsTwo + ouncesTwo / 16;
+    const rate = ((totalTwo - total) / total).toFixed(2);
```

Comment:
🚫 **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'rate' is assigned a value but never used.